### PR TITLE
Add landing page, mission section, and update theme

### PR DESCRIPTION
**Pull request by indhu-gunda**
capture (4)

I based the mission description on the design doc. Let me know if you guys wanna change that or the short phrase under luminart.

### CODE REVIEW

File: frontend/src/components/dashboard/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    position: 'relative',
+    width: '100%',
+  },
+  bannerText: {
+    left: '30%',
+    position: 'absolute',
+    top: '40%',
+    transform: 'translate(-50%,-50%)',
+    width: '100%',
+    textAlign: 'center',
```
Comment (katch00) [inline frontend/src/components/dashboard/LandingPage.js:27]: style nit: alphabetical order

File: frontend/src/components/dashboard/Dashboard.js [hunk 2]
```diff
@@ -108,123 +139,117 @@ export default function Dashboard() {
...
+                color="inherit"
+                aria-label="open drawer"
+                onClick={handleDrawerOpen}
+                edge="start"
+                className={clsx(classes.menuButton, open && classes.hide)}
+              >
+                <MenuIcon />
+              </IconButton>
+              <Typography variant="h6" noWrap>
+                    Header
```
Comment (isabelsang) [inline frontend/src/components/dashboard/Dashboard.js:163]: We can probs change this to Luminart now :)

File: frontend/src/components/dashboard/LandingPage.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  },
+}));
+
+export default function Banner() {
```
Comment (isabelsang) [inline frontend/src/components/dashboard/LandingPage.js:31]: Is this supposed to be `export default function LandingPage()` ? I could be mistaken

Comment (indhu-gunda) [reply frontend/src/components/dashboard/LandingPage.js:31]: it is, I'm not sure how it even worked properly with that bug

File: frontend/src/components/dashboard/LandingPage.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        >
+          Luminart
+        </Typography>
+        <Typography
+          variant="h4"
+          component="h2"
+          color="textPrimary"
+          gutterBottom
+        >
+          Shining a light on a new way to experience art for everyone.
```
Comment (isabelsang) [inline frontend/src/components/dashboard/LandingPage.js:56]: I like the idea of this but feel like it is a little wordy.  I am having a hard time wording a blurb myself.  I was thinking simplifying it to be "Shining a light on a new way to experience art" or even "Shining a new light on art" but I don't think those get across that the point that our website is universally accessible.  So maybe something like "Shining a light for all to enjoy art"? Still not perfect but just wanted to share a few thoughts, feel free to edit any of those, and we can brainstorm together later if you would like

**Pull request merged**